devtools/conda-envs/test_env.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   - torchvision
   - openff-units
   - torchmetrics>=1.4
-  - pint
+  - pint=0.23
```

Comment:
is this pin necessary?

Reply:
This is necessary on my local setup.

Reply:
I was also having issues with openff-units and pint, not sure if that's related. I had to pin openff-units>=0.2.0

Reply:
we don't see this in CI, that's strange

Reply:
@chrisiacovella , have you seen this?